tests/README.md

```diff
@@ -0,0 +1,25 @@
+Install the dependencies using
+
+`pip install -r requirements/CI/requirements.txt`
+
+SLiM recipes used for testing are in `examples/`: the specifications for which
+tests to run on them are specified in `slim_recipe_specs.py`
+
+To run the entire test suite: 
+
+```
+python3 -m pytest
+```
+
+To restrict to individual tests, the syntax is:
+
+```
+python3 -m pytest tests/test_tree_sequence.py::test_slim_generation
+```
+
+
+To restrict to specific recipes in the examples dir, the syntax is:
+
+```
+python3 -m pytest -k nonWF_recipe
```

Comment:
very nice. But, what is `nonWF_recipe`? There aren't any example slim scripts named this?

Reply:
Seems to be to me: https://github.com/tskit-dev/pyslim/pull/131/files#diff-5f283adae7de0cdcc50a5773f9873d8b62ec0d5e0313f6bdbda6ec4157671dbd

Reply:
I guess it's obvious that these can be combined?

`python3 -m pytest tests/test_tree_sequence.py::test_slim_generation -k nonWF_recipe`

Reply:
Ah, ok. But, that's `recipe_nonWF`, not `nonWF_recipe`.

Reply:
oh wait maybe you renamed all the recipes?

Reply:
I had to change the names and put `_recipe` at the end, not `recipe_` at the front (discussed in the initial comment).

Reply:
What's the motivation for that? I thought you were removing the repetitive `recipe_` at the start... but it's at the end, now?

Reply:
The `-k` flag matches its arg anywhere in the test string. Since there were some recipes named `recipe_nonWF.slim` and `recipe_nonWF_early.slim`, then running `pytest -k recipe_nonWF` would run both. If you put `_recipe` on the end then that serves as a useful terminator to stop this.

Reply:
I guess the other way would be to require the `.slim` prefix to be passed: `pytest -k recipe_nonWF.slim`, and then I would have to cut off the `.slim` before checking for the settings in `slim_recipe_specs.py`. Perhaps you prefer this? It's a bit more fiddly though.

Reply:
Hmm, I forgot that the prefix isn't unique either. So we have `WF_early_recipe` and `nonWF_early_recipe`, both of which will now match with `-k WF_early_recipe`. Argh. Looks like we need `recipe_` back on the front and `.slim` on the end too.

Reply:
arg, that's fiddly. But it makes it a bit more clear to have `.slim` on the end anyhow, IMO.